.gitlab/pnnl/README.md

```diff
@@ -0,0 +1,86 @@
+# How to connect GitHub to a PNNL GitLab (push) Mirror
+
+To run on our HPC clusters at PNNL while hosting our code base on GitHub, we utilize PNNL's CI/CD services on GitLab. 
+
+With GitLab premium, integration between GitHub and GitLab is included. This is how we have architected a solution with the base level GitLab offering. 
```

Comment:
```suggestion
With GitLab premium, integration between GitHub and GitLab is included. The tier of GitLab that we have does not and did not initially support this. While switching to premium would alleviate some of the burden here, our technical requirements expanded having to test on more than one cluster. This is how we have architected a solution with the base level GitLab offering. 
```

Reply:
The key to this suggestion is that we trigger the CI once, and then we need to report multiple build status pieces of information anyway...